docker/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+mysqldata:
+  image: mysql
+  command: true
```

Comment:
I was getting `json: cannot unmarshal bool into Go value of type string`

Changing it to `/bin/true` seems to have fixed it.